profiling-ffi/Cargo.toml

```diff
@@ -14,8 +14,9 @@ license.workspace = true
 crate-type = ["staticlib", "cdylib"]
 
 [features]
-default = ["cbindgen"]
+default = []
```

Comment:
I think that having a default that does not generate can make sense.
@paullegranddc wdyt ?